plugins/importexport/doaj/classes/PubObjectCache.inc.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * @file plugins/importexport/doaj/classes/PubObjectCache.inc.php
```

Comment:
Is this a duplicate of the PubObjectCache class used elsewhere? If it's identical, better ot use a symlink rather than duplicating the class entirely.